content/en/01-about-pixie/04-architecture.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Architecture"
+metaTitle: "About Pixie | Architecture"
+metaDescription: "What Pixie deploys to your cluster and what each component does."
+order: 4 
+---
+
+## Main Components 
```

Comment:
I think this doc needs a bit more context. Specifically I think we need to clarify

1. How does Pixie Cloud fit in?
2. What is deployed to my Kubernetes cluster?

I think we need a preamble section setting the context before diving into the components, and ideally a diagram (can be the same one we already use).

"Pixie's deploy" is a bit vague because most users are OSS users and they will deploy both cloud and Vizier to their cluster. That being said, we should still word the pre-amble in such a way that it applies